stack/base/llama-stack-distribution.yaml

```
@@ -7,6 +7,13 @@ spec:
   replicas: 1
   server:
     containerSpec:
+      resources:
+        requests:
+          cpu: "250m"
+          memory: "500Mi"
+        limits:
+          cpu: "2"
+          memory: "12Gi"  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)